plugins/module_utils/network/nxos/argspec/logging_global/logging_global.py

```diff
@@ -155,7 +156,7 @@ class Logging_globalArgs(object):  # pylint: disable=R0903
                         "server": {"type": "str"},
                         "level": {"type": "int"},
                         "facility": {"type": "str"},
-                        "port": {"type": "str"},
+                        "port": {"type": "int"},
```

Comment:
in case of iosxr port has default value as "default". Do we have any default value for port in NXOS?

Reply:
The `default` value doesn't appear in running-config for NX-OS.

Reply:
Nope. All possible values are integers.

Reply:
okay